src/gc/gcee.cpp

```diff
@@ -392,7 +392,7 @@ size_t GCHeap::GetNow()
 
 void ProfScanRootsHelper(Object** ppObject, ScanContext *pSC, DWORD dwFlags)
 {
-#if defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
+#if defined(GC_PROFILING) && defined(FEATURE_EVENT_TRACE) 
```

Comment:
ScanRootsHelper defined in proftoeeinterfaceimpl.cpp is under define GC_PROFILING https://github.com/dotnet/coreclr/blob/4add7616fae2fe83bde5884e11c761d33a20cffe/src/vm/proftoeeinterfaceimpl.cpp#L921